Inquirer function params

RobinKnipe opened the pull request:
This PR addresses issue #189 
Call the `inquirer` functions with the expected `answers` hash parameter.

crutchcorn commented:
The code/tests look good here! Admittedly, though, I'm still not sure what this change means for consumers of `node-plop`. I'll likely merge either way, but want to make sure I fully understand the implications of the change

RobinKnipe commented:
Hi @crutchcorn - I think your reply was quicker than the circle-ci tests (which I've fixed)! I've also added a similar test to check the validate function receives the answers hash too. The inquirer function definitions can be found [here](https://github.com/SBoudrias/Inquirer.js#question) FYR.
